src/app.js

```diff
@@ -1,6 +1,10 @@
+import createBtn from './btn';
+
+const root = document.querySelector('#app');
+const btn = createBtn('play the game');
+root.appendChild(btn);
```

Comment:
This should be inside of app function body

Reply:
it won't work in the function body

Reply:
it won't work

Reply:
yes it will, but you have to call that function (it was like this before)